src/NzbDrone.Core/Music/RefreshAlbumService.cs

```diff
@@ -67,11 +67,12 @@ public void RefreshAlbumInfo(Artist artist, IEnumerable<Album> remoteAlbums)
                     albumToUpdate.CleanTitle = album.CleanTitle;
                     //albumToUpdate.TrackNumber = album.TrackNumber;
                     albumToUpdate.Title = album.Title ?? "Unknown";
+                    albumToUpdate.CleanTitle = Parser.Parser.CleanArtistTitle(albumToUpdate.Title);
                     //albumToUpdate.AlbumId = album.AlbumId;
                     //albumToUpdate.Album = album.Album;
                     //albumToUpdate.Explicit = album.Explicit;
                     albumToUpdate.ArtistId = artist.Id;
-                    albumToUpdate.Path = artist.Path + album.Title;
+                    albumToUpdate.Path = artist.Path + "\\" + album.Title;
```

Comment:
Using BuildAlbumPath in FileNameBuilder will first check to see if the user has AlbumFolders enabled for that artist, it album folders is not enabled it returns the artist path as the album path, if it is then it builds out the path per the rename conventions in settings. 

Reference up to line 62 of this file, this should be the correct way to get path I believe. If not we are just hard coding the path to always be album title and giving user no control.